website/docs/advanced-concepts/user-provisioning-group-sync.mdx

```diff
@@ -27,7 +27,8 @@ This page shows how to configure Appsmith for SCIM to enable the automatic provi
 
 ## Prerequisites
 
-- A self-hosted Appsmith instance. See the [installation guides](/getting-started/setup/installation-guides) for deploying Appsmith.
+* A self-hosted Appsmith instance. See the [installation guides](/getting-started/setup/installation-guides) for deploying Appsmith.
+* Ensure that the Identity Provider (IDP) can connect with Appsmith.
```

Comment:
@sharat87 @pratapaprasanna : do we need to add any explicit information to check for access as part of the documentation? Or this much is good enough?

Reply:
@trishaanand - should we go ahead and merge this? If needed, we can make incremental changes.

Reply:
Sure. Please go ahead.